docker-compose.yml

```diff
@@ -1,306 +1,441 @@
+    container_name: image-goaccess
+    image: allinurl/goaccess:1.6.1
+    restart: unless-stopped
+    command: /srv/logs/image-a11y-mcgill-ca-access.log --log-format=COMBINED --date-format='%d/%b/%Y' --time-format='%T' --no-global-config --real-time-html -o /srv/report/report.html --persist --restore
+    volumes:
+      - website-logs:/srv/logs:ro
+      - ./goaccess_report:/srv/report
+
+# start - common services
+  orchestrator:
+    profiles: [pegasus, unicorn, default]
+    image: ghcr.io/shared-reality-lab/image-orchestrator:${REGISTRY_TAG}
+    restart: unless-stopped
+    environment:
+      - PARALLEL_PREPROCESSORS=ON
+      - STORE_IMAGE_DATA=ON
+    env_file:
+      - ./config/express-common.env
+    group_add:
+      # This should be set in an environment variable or .env
+      - ${DOCKER_GID}
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+      - user-logs:/var/log/IMAGE
+
+  espnet-tts:
+    profiles: [pegasus, unicorn, default]
+    image: ghcr.io/shared-reality-lab/image-service-espnet-tts:${REGISTRY_TAG}
+    restart: unless-stopped
+    environment:
+      - TORCH_DEVICE=cuda
+    ports:
+      - "8001:80"
+    deploy:
+      resources:
+        reservations:
+          devices:
+          - driver: nvidia
+            capabilities: ["gpu", "utility", "compute"]
+
+  espnet-tts-fr:
+    profiles: [pegasus, unicorn, default]
+    image: ghcr.io/shared-reality-lab/image-service-espnet-tts-fr:${REGISTRY_TAG}
+    restart: unless-stopped
+    environment:
+      - TORCH_DEVICE=cuda
+    deploy:
+      resources:
+        reservations:
+          devices:
+            - driver: nvidia
+              capabilities: ["gpu", "compute", "utility"]
+
+  multilang-support:
+    profiles: [pegasus, unicorn, default]
+    image: ghcr.io/shared-reality-lab/image-service-translation:${REGISTRY_TAG}
+    restart: unless-stopped
+    deploy:
+      resources:
+        reservations:
+          devices:
+            - driver: nvidia
+              capabilities: ["gpu", "compute", "utility"]
+
+  autour-preprocessor:
+    profiles: [pegasus, unicorn, default]
+    image: ghcr.io/shared-reality-lab/image-preprocessor-autour:${REGISTRY_TAG}
+    restart: unless-stopped
+    env_file:
+      - ./config/maps.env
+    labels:
+      ca.mcgill.a11y.image.preprocessor: 1
+      ca.mcgill.a11y.image.port: 5000
+
+  content-categoriser:
+      profiles: [pegasus, unicorn, default]
+      image: ghcr.io/shared-reality-lab/image-preprocessor-content-categoriser:${REGISTRY_TAG}
+      restart: unless-stopped
+      labels:
+        ca.mcgill.a11y.image.preprocessor: 1
+        ca.mcgill.a11y.image.port: 5000
+      deploy:
+        resources:
+          limits:
+            memory: 2g
+
+  graphic-tagger:
+    profiles: [pegasus, unicorn, default]
+    image: ghcr.io/shared-reality-lab/image-preprocessor-graphic-tagger:${REGISTRY_TAG}
+    restart: unless-stopped
+    env_file:
+      - ./config/azure-api.env
```

Comment:
Since these other configs are not checked in, likely need some documentation or an example file, so people know what it needs to contain?

Reply:
I don't think that this is how gitignore works...I think that once a file is tracked, the ignore is not applied to it. Hope I'm wrong here though, since this would be an ideal solution.